DESCRIPTION

```diff
@@ -18,9 +18,12 @@ RoxygenNote: 7.2.3
 Imports:
     BiocManager,
     Biostrings,
-    tidyverse,
```

Comment:
@jakekrol, why are you loading all the individual libraries again here?

Reply:
I did this to import only the necessary subset of dependencies from tidyverse, instead of all.

This is the general consensus from what I've seen online in the community. 

Tidyverse itself, suggests against depending on the meta package: https://www.tidyverse.org/blog/2018/06/tidyverse-not-for-packages/

Reply:
Hmm, I see. I never thought of CRAN -- so, never considered this. OK, let's use tidyverse for scripts/r/qmds & individual ones for imports. I always feared I'd miss something -- so, I loaded them all up just to be sure! This definitely makes sense, esp. when starting from scratch. 👍🏼